app/recurring_expenses/views.py

```diff
@@ -123,6 +123,52 @@ def create_recurring_expense(request):
     return render(request, "recurring_expenses/new.html", {"form": form})
 
 
+@login_required
+def export_recurring_expenses(request):
+    if "csv" in request.GET["format"]:
+        filename = "recurring_expenses.csv"
+        response = HttpResponse(
```

Comment:
FWIW we could try https://docs.djangoproject.com/en/4.2/ref/request-response/#fileresponse-objects in the future